time_execution/backends/threaded.py

```diff
@@ -24,6 +24,7 @@ def __init__(self, backend, backend_args=None, backend_kwargs=None,
             backend_args = tuple()
         if backend_kwargs is None:
             backend_kwargs = dict()
+        self.parent_thread = threading.current_thread()
```

Comment:
OK. I think that's fine. But then you can add a proper test by creating a thread to instantiated backend.